test/e2e-tests.sh

```diff
@@ -34,8 +34,12 @@ fi
 
 header "Setting up environment"
 
+install_spire
```

Comment:
Since this is an alpha feature for now, I suppose we should only install SPIRE on the the job with alpha features enabled?

Reply:
Yes I will incorporate that